built-with-eleventy/-1R2-Wp5Q9.json

```diff
@@ -1,6 +1,9 @@
 {
   "url": "https://www.ineliagestion.com/",
-  "opened_by": "madrilene",
+  "opened_by": "lene",
```

Comment:
I think this `opened_by` is a reference to your GitHub user ID, so probably needs to stay as "madrilene" since `@lene` is somebody else.

Reply:
Ah! Thank you,I'll redo that!!

Reply:
I changed that back now!
What I am trying to achieve is getting the article(s), starters and website projects all in the same author page (https://www.11ty.dev/authors/madrilene/), which could also be called https://www.11ty.dev/authors/lene/, but if that's difficult, madrilene is fine of course.